### PR TITLE
[startup]  Fixes issue with /var/platform directory not created

### PR DESCRIPTION
**Pull request by hpersh**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: files/image_config/platform/rc.local [hunk 1]
```diff
@@ -328,10 +328,10 @@ if [ -f $FIRST_BOOT_FILE ]; then
         mv /host/grub.cfg /host/grub/grub.cfg
     fi
 
+    # Create dir where following scripts put their output files
+    mkdir -p /var/platform
```
Comment (lguohan) [inline files/image_config/platform/rc.local]: why do we create it?

Comment (anand-kumar-subramanian) [reply files/image_config/platform/rc.local]: We have created this to support system related queries and to support platform specific validations. We will remove this dependency in the future once the system calls are serviced through host dbus implementation.

**Comment by jleveque** [comment]
Retest vs please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vs please

**Comment by jleveque** [comment]
Retest vsimage please

**Pull request merged**